### PR TITLE
Add pom.xml for benchmarks

### PR DESCRIPTION
**Pull request by mfussenegger**
Follow up to https://github.com/crate/crate/pull/13651

### CODE REVIEW

File: devs/docs/benchmarks.rst [hunk 1]
```diff
@@ -2,18 +2,29 @@
...
 ==========
 
-Microbenchmarks are written using `JMH`_. They can be executed by using the
-``:benchmarks:run``::
+Microbenchmarks are written using `JMH`_. To execute them, first build the JAR::
 
-    $ ./gradlew :benchmarks:run
+    $ ./mvnw -T 1C package -DskipTests=true
+
+The run the JAR::
```
Comment (BaurzhanSakhariev) [inline devs/docs/benchmarks.rst:9]: Minor: then run

File: benchmarks/pom.xml [hunk 2]
```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.crate</groupId>
+    <artifactId>crate</artifactId>
+    <version>5.5.0</version>
```
Comment (BaurzhanSakhariev) [inline benchmarks/pom.xml:9]: Do I get it right, that we will be changing this version on each feature release? Is it an option to create some variable and refer to it everywhere we use hardcoded version?

**Review by BaurzhanSakhariev — APPROVED** [review]
Spot a minor typo and added a question, LGTM otherwise

File: benchmarks/pom.xml (hunk 2 above)
Comment (mfussenegger) [reply benchmarks/pom.xml:9]: Maven provides a command to bump it across all modules: `mvn versions:set -DnewVersion=5.6.0`. But good point. I'll need to update the release docs/steps

**Pull request merged**